packaging/bundle-validation/Dockerfile-base

```diff
@@ -47,3 +49,44 @@ RUN wget https://archive.apache.org/dist/spark/spark-$SPARK_VERSION/spark-$SPARK
     && tar -xf $WORKDIR/spark-$SPARK_VERSION-bin-hadoop$SPARK_HADOOP_VERSION.tgz -C $WORKDIR/ \
     && rm $WORKDIR/spark-$SPARK_VERSION-bin-hadoop$SPARK_HADOOP_VERSION.tgz
 ENV SPARK_HOME=$WORKDIR/spark-$SPARK_VERSION-bin-hadoop$SPARK_HADOOP_VERSION
+
+
+# Utilities bundles
+RUN wget https://repo1.maven.org/maven2/org/apache/hudi/hudi-utilities-bundle_$SCALA_VERSION/0.11.0/hudi-utilities-bundle_$SCALA_VERSION-0.11.0.jar -P "$WORKDIR" 
+ENV UTILITIES_BUNDLE_0_11_0=$WORKDIR/hudi-utilities-bundle_$SCALA_VERSION-0.11.0.jar
```

Comment:
we talked about upgrade/downgrade test scenario which should only be run for release branch, and for this reason, we should parameterize the versions to pull based on the release branch name like if the branch name is `release-0.13.0` -> downloading `0.11.{latest}` and `0.10.{latest}` for testing. 

Another design we should follow here: we build infra with docker, which is generic and re-usable, and make testing artifacts mounted via volumes, for easy switching, flexible with testcases. So by this, we should actually download jars from GH actions job itself and mount those to container. Because this is only for release branch tests, it won't "waste" a lot in downloading. If to optimize downloading jars, we can even explore GH feature to see if we can put jars in GH's own artifactory for probably cached downloading